frontend/src/actions/parcelsActions.ts

```diff
@@ -13,6 +13,7 @@ export enum PropertyTypes {
 export interface IProperty {
   id: number | '';
   propertyTypeId?: PropertyTypes;
+  agencyId?: number | '';
```

Comment:
I don't like this implementation, but based current requirements properties not owned by the user's agency or sub-agencies are only allowed to share certain information.  The owning agency of the property is not currently supposed to be shared.